src/ProductConstructionService/ProductConstructionService.WorkItems/WorkItemConfiguration.cs

```diff
@@ -40,7 +41,20 @@ public static void AddWorkItemQueues(this IHostApplicationBuilder builder, Defau
             builder.Configuration.GetRequiredValue(WorkItemQueueNameConfigurationKey);
         builder.Services.Configure<WorkItemConsumerOptions>(
             builder.Configuration.GetSection(WorkItemConsumerOptions.ConfigurationKey));
-        builder.Services.AddHostedService<WorkItemConsumer>();
+
+        var consumerCountConfig = builder.Configuration.GetRequiredValue(WorkItemConsumerCountConfigurationKey);
+        if (int.TryParse(consumerCountConfig, out var consumerCount))
```

Comment:
I'd maybe make this just `Parse` so that it's obvious this needs to be configured. Otherwise it might be quite hard to figure out why there's no listeners.